packages/default-theme/components/SwErrorsList.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div v-if="list" class="errors-list-component">
+    <SwAlert :message="alertMessage" type="danger" />
+
+    <ul class="list">
+      <li v-for="message in list" :key="message" class="item">{{ message }}</li>
+    </ul>
+  </div>
+</template>
+
+<script>
+import SwAlert from "@shopware-pwa/default-theme/components/atoms/SwAlert"
+
+export default {
+  name: "SwErrorsList",
+  components: { SwAlert },
+  props: {
+    list: {
+      type: Array,
+      default: null,
+    },
+  },
+  data() {
+    return {
+      alertMessage: "Encountered problems:",
```

Comment:
not sure, but the translation should come from ` this.$i18n.t(` function

Reply:
I removed that part of code :+1: